rest-api/src/main/java/io/quarkus/qe/RepositoryResource.java

```diff
@@ -36,6 +31,15 @@ public Response get(@PathParam("id") Long id) {
         }
     }
 
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("/")
+    public Response getAll(@QueryParam("from") @DefaultValue("0") int from, @QueryParam("to") @DefaultValue("20") int to) {
```

Comment:
We should maintain the parameter names as in the documentation: https://quarkus.io/guides/rest-data-panache#query-parameters. Also, adding the sort is a nice thing to have but not required at this point.